packages/hashcat/Makefile.patch

```diff
@@ -0,0 +1,40 @@
+--- ./src/Makefile
++++ ../src/Makefile
+@@ -4,7 +4,7 @@
+ ##
+ 
+ SHARED                  := 0
+-DEBUG                   := 0
++DEBUG                   := 1
+ PRODUCTION              := 1
+ 
+ ##
+@@ -44,7 +44,7 @@
+ ##
+ 
+ DESTDIR                 ?=
+-PREFIX                  ?= /usr/local
++PREFIX                  ?= /data/data/com.termux/files/usr
```

Comment:
You could write @TERMUX_PREFIX@ instead of /data/data/com.termux/files/usr, build-package.sh automatically sed's it.

Reply:
`@` should be there ri8?

Reply:
Yeah, see for example: https://github.com/termux/termux-packages/blob/master/packages/moria/source-config.h.patch